tests/misc/tdllvar.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-disabled: true
+    disabled: true
```

Comment:
Created around 2014-04 in a commit that references #1075
Disabled around 2018-11 for CI sake?

Reply:
Disable, I guess, not sure

Reply:
Keeping this disabled for now.